developer-docs-site/docs/concepts/base-gas.md

```diff
@@ -273,7 +273,7 @@ As of the time of this writing, by far the most expensive instruction gas operat
 Loading a constant costs 650 gas units (.065 octals), borrow operations cost 500 units (0.05 octals), reading or writing to a reference costs 200 gas units (0.02 octals), and loading a `u128` on the stack costs 80 gas units (0.008 octals).
 Hence pass-by-value is less expensive than pass-by-reference for a primitive type like `u64`, but more expensive for a larger data structure like a 40-byte `struct`.
 
-Notably, instruction gas is completely dwarfed by storage costs; there is technically an incentive to reduce the number of function calls in a program, for example, engineering efforts are more effectively dedicated to writing modular, decomposed code that is geared toward reducing storage gas costs, rather than attempting to write repetitive code blocks with fewer nested functions (in nearly all cases).
+Notably, instruction gas is completely dwarfed by storage costs; there is technically an incentive to reduce the number of function calls in a program, for example, but engineering efforts are more effectively dedicated to writing modular, decomposed code that is geared toward reducing storage gas costs, rather than attempting to write repetitive code blocks with fewer nested functions (in nearly all cases).
```

Comment:
Suggested minor rewrite (I only looked at this paragraph): 

Notably, instruction gas is completely dwarfed by the storage costs. For example, there is technically an incentive to reduce the number of function calls in a program, but engineering efforts are more effectively dedicated to writing modular, decomposed code that is geared toward reducing storage gas costs, rather than attempting to write repetitive code blocks with fewer nested functions (in nearly all cases).

Reply:
The suggested minor rewrite looks good to me, but I only just saw this and it looks like the commit has already been merged. I yield to you on the next PR and any other minor rewrites, thanks!